packages/styles/scss/themes/expressive/components/_inline-notifications.scss

```diff
@@ -14,6 +14,7 @@
 @mixin inline-notifications-expressive {
   .#{$prefix}--inline-notification {
     .#{$prefix}--inline-notification__action-button.#{$prefix}--btn--ghost {
+      margin-top: 7px;
```

Comment:
Adding @annawen1 here to get eyes on, but wondering if this will hold water at all if users try to zoom the browser.

Reply:
did we ever come to a solution for zooming cases in the QA calls? I don't remember if we did.